### DISCUSSION TITLE
Module not found error in Jupyter notebook while open3d can be seen installed in base as well as VE

### DISCUSSION THREAD

**Discussion by Inshu32** [isl-org/Open3D#6417] [Q&A]
I installed open3d in base and my virtual environment, but when I open Jupyter notebook through my Linux terminal (debian based) and try to import open3d it says 'Module not found*. I installed it by building from source so that I have correct dependencies. But still snot working. I am working with tensorflow and Keras CV in the same virtual environment, if that messes up any thing ? I tried both kernels  - python as well as my virtual environment in jupyter notebook. Still its the same error.

**Comment by Inshu32** [comment]
![Untitled](https://github.com/isl-org/Open3D/assets/145352703/590670c7-23e1-443a-8b62-0f4436773e77)

**Reply by saurabheights** [reply]
@Inshu32 
Did you activate conda environment before running `jupyter notebook`? 

Also, try running directly via `python3 -c "import open3d"` first. Maybe the issue is in incorrect installation.

**Reply by Inshu32** [reply]
yes I did activate conda environment. @saurabheights Thanks & ya will try python3 -c "import open3d and get back.

**Reply by Inshu32** [reply]
Hi is open3d not compatible with python 3.11 ?

**Reply by saurabheights** [reply]
@Inshu32 No. There is some work thats going on to upgrade Open3d to version 3.11 however there is no ETA on it.
